src/api/mediators.coffee

```diff
@@ -53,6 +55,18 @@ exports.addMediator = ->
 
   try
     mediator = this.request.body
+
+    if mediator?.endpoints?[0]?.host?
+      mediatorHost = mediator.endpoints[0].host
+    else
+      mediatorHost = 'unknown'
+
+    # audit mediator start
+    audit = atna.appActivityAudit true, mediator.name, mediatorHost, 'root'
```

Comment:
to be fair, we don't actually know that it's the root user. Not sure if there's a way to pick up the user dynamically - couldn't see anything from a quick search - maybe rather call it `system` instead? just to make it a clearer